packages/video_player/video_player/example/ios/Flutter/Flutter.podspec

```diff
@@ -0,0 +1,18 @@
+#
```

Comment:
Not sure how this got in here, can you remove this and project.pbxproj files from the PR?